_data/fellows/ballance-robert.yml

```diff
@@ -0,0 +1,40 @@
+id: ballance-robert
+name: Dr. Robert Ballance
+img: ballance-robert-headshot.jpg
+year: 2016
+hometown: "Santa Fe, NM"
+region: west
+bio: "
+Computer scientist; catalyst; strategist; manager of high-tech teams and large-scale HPC systems.  Ph.D. from UC, Berkeley."
+bio_full: "
+Bob works in the intersection of collaborative teams, system design,
+large-scale operations, user support, service delivery, and software
+development where interactions are frequent and learning is
+continuous. As a practicing computer scientist, he helps people and
+organizations to mesh with computing systems. His job is often
+linguistic. It's about getting the team to agree on a common language
+so that they can move forward together. He's served in multiple roles:
+graduate student, professor, researcher, developer, entrepreneur,
+consultant, manager of high-performance computing (HPC) systems,
+leader of high-tech teams, IT strategist.
+<br></br>
```

Comment:
Nice use of HTML!

There's no closing line break tag though — this should actually just be:

```HTML
<br><br>
```